packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -16,7 +16,7 @@ import 'material_state.dart';
 import 'theme.dart';
 import 'theme_data.dart';
 
-const Duration _kTransitionDuration = Duration(milliseconds: 200);
+const Duration _kTransitionDuration = Duration(milliseconds: 167);
```

Comment:
nit: could you add a comment to explain the provenance of this value? Something like "value extracted from <link>"